models/balancer/ethereum/balancer_vebal_slopes.sql

```diff
@@ -0,0 +1,149 @@
+{{
+    config(
+        alias='vebal_slopes',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_time', 'wallet_address'],
```

Comment:
Do you think `block_number` may be a better component of unique_key(rather than `block_time`)? As block_number is int-typed, and easier for join.

Reply:
Since it makes no change for the query and both should be unique, I believe it may be a better compoment. Just commited this change.